### ISSUE TITLE
Older version of QGIS

### ISSUE THREAD

**Issue by Jackson-R18** [qgisred/QGISRed#133]
Can you download older versions of QGIS? I am trying to download version 3.26.3 but can't find how to anywhere

**Comment by fmartine** [comment]
Dear Jackson_R18,

See my reply to your recent issue #132

**Comment by fmartine** [comment]
You are right. I have tried to run QGISRed with the latest QGIS version 3.36.3 and it gives again a format error in a function call.

While we solve the problem, you can use the latest stable version 3.34.7 LTR, accessible from the QGIS web page, in which QGISRed version 0.16.11 works correctly.
